enterprise-suite/scripts/lbc.py

```diff
@@ -98,15 +98,15 @@ def run(cmd, timeout=None, stdin=None, show_stderr=True):
     finally:
         if timer != None:
             timer.cancel()
-        return stdout, returncode
+        return returncode, stdout, stderr
```

Comment:
Would it be simpler to combine stdout and stderr?

Reply:
Another option to make this less cumbersome is to return a dictionary.

Reply:
I'd rather not combine them.  Gonna leave alone for now but I agree it could be improved.